src/othermbc.md

```diff
@@ -40,70 +41,75 @@ nonsensical.
 
-First, unlike a usual MBC, it switches the whole 32 kiB ROM area instead
-of just the $4000-$7FFF area. If you want to use the interrupt vectors
+First, unlike a usual MBC, it switches the whole 32 KiB ROM area instead
+of just the $4000-$7FFF area. Therefore, if you want to use [the interrupt vectors](<#Interrupt Handling>)
 with this cart, you should duplicate them across all banks.
 Additionally, since the initial state of the '377 can't be guaranteed,
 the ROM header and some code for switching to a known bank should also
 be included in every bank. This also means that the Wisdom Tree mapper
-could be used as a multicart mapper for 32 kiB ROMs, assuming there was
+could be used as a multicart mapper for 32 KiB ROMs, assuming there was
 enough ROM space in each bank for some small initialization code, and
 none of the ROMs wrote to the $0000-$7FFF area. For example, if the
 last 5 bytes of all banks are unused, games can be patched as follows:
 
-    ; At $0100 in all banks but the first
-      nop
-      jp $7FFB
-
-    ; At $7FFB in all banks
-      ld hl, $0100
-      ld [hl], a
-      jp hl
-
-Second, because the '377 latches data on the *positive* edge, and the
-value on the Game Boy data bus is no longer valid when the positive edge
-of the write pulse arrives, the designer of this mapper chose to use the
+```rgbasm
+; At $0100 in all banks but the first
+    nop
+    jp $7FFB
+```
+```rgbasm
+; At $7FFB in all banks
+    ld hl, $0100
+    ld [hl], a
+    jp hl
+```
+
+Second, because the '377 latches data on the *positive* write pulse edge,
+and the value on the Game Boy data bus is no longer valid when the
+positive edge arrives, the designer of this mapper chose to use the
 A7-A0 address lines for selecting a bank instead of the data lines.
 Thus, the value you write is ignored, and the lower 8 bits of the
 address is used. For example, to select bank $XX, you would write any
 value to address $YYXX, where $YY is in the range $00-$7F.
 
-An emulator can detect a ROM designed for Wisdom Tree mapper in one of
+An emulator can detect a ROM designed for the Wisdom Tree mapper in one of
 two ways:
 
--   ROM contains "WISDOM TREE" or "WISDOM\\x00TREE" (the space can
-    be $20 or $00), $0147 = $00, $0148 = $00, size \> 32k. This
-    method works for the games released by Wisdom Tree, Inc.
--   $0147 = $C0, $014A = $D1. These are the values recommended by
-    beware for 3rd party developers to indicate that the ROM is
-    targeting Wisdom Tree mapper hardware. (See below.)
+- [ROM title](<#0134-0143 - Title>) is "WISDOM TREE" (the space may be a
+  $00 NUL character instead), $0147 = $00, $0148 = $00, size \> 32k.
+  This method works for the games released by Wisdom Tree, Inc.
+- $0147 = $C0, $014A = $D1. These are the values recommended by
+  beware for 3rd party developers to indicate that the ROM is
+  targeting Wisdom Tree mapper hardware. (See below.)
 
 ## Magic values for detection of multicarts in emulators
```

Comment:
Why is there a chapter about detecting this type of multicart when it has just been stated how to detect it? This information should be merged

Reply:
On one hand there's documentation of the existing ROMs, and OTOH there's documentation for homebrew developers seeking to use those mappers (for any reason). There's also emulator developers, who still read Pan Docs, who may want to implement detection, and for whom documenting an agreed-upon heuristic is better than letting subtly incompatible variations proliferate.

I agree that it should be moved elsewhere as an actual standard, but this is not the scope of this PR.

Reply:
Whatever their scope, this information should be in the same place, and not splitted in "for emu dev" vs "for homebrew dev": it's the same suggestion and splitting it only creates confusion.

> It is proposed that homebrew targeting this mapper should [...]. 

Emulator developers can read this kind of information with no problems, and additional notes or info boxes can be appended to let them know additional/proposed information